foam_wrapper/foam_wrapper.py

```diff
@@ -0,0 +1,228 @@
+""" foam_wrapper module
+
+Wrapper module for OpenFOAM using pythonFlu -python interface
+NOTE: at a moment pythonFlu supports only up to 2.0.1 -version of OpenFOAM
+
+"""
+
+from Foam import ref, man
+from simphony.cuds.mesh import Mesh, Point, Face, Edge, Cell
+#import inspect
+#import copy
+#from Foam.OpenFOAM import fileName, word
+from collections import defaultdict
+#from Foam.finiteVolume import volScalarField
+
+
+class Foam_wrapper(object):
+
+    def __init__(self, model):
+        self.model = model
+
+    def has_common_vertex(self, facePoints0, facePoints1):
```

Comment:
This function does not have to be inside the FoamWrapper class, I would suggest to move it outside as a utility function